.eslintignore

```diff
@@ -11,5 +11,4 @@ bin/
 config/
 public/
 templates/
-tests/
```

Comment:
@chirimoya Seems like this did not only ignore the top level `tests` directory, also explains why it was faster :see_no_evil: